packages/integration-tests/test/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// tslint:disable-next-line:no-var-requires
+const testRunner = require('vscode/lib/testrunner');
```

Comment:
This is not the actual test runner. This is just a convenience mechanism to use the vscode/bin/test to download the vscode binaries for different platforms. The actual tests are in the scenarios folder and run by main.js -> mocha-runner.js